### PR TITLE
Add class price field

### LINKED ISSUE #116: Add the money per class field into Person
A field for `money per class` should be added into the `Person` class in order to determine how much increment there should be when a person is marked as Present for a class.

### PR DESCRIPTION
**Pull request by Echomo-Xinyu**
Fixes #116 

- Add the field of `money per class` into `Person` as part of the `add mark command` task.
- Relevant methods have been modified accordingly;
- Relevant test cases have been updated accordingly;
- UI has been updated to display the `money per class` field

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -116,11 +118,12 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Class updatedClassDateTime = editPersonDescriptor.getAClass().orElse(personToEdit.getAClass());
         Money updatedMoneyOwed = editPersonDescriptor.getMoneyOwed().orElse(personToEdit.getMoneyOwed());
         Money updatedMoneyPaid = editPersonDescriptor.getMoneyPaid().orElse(personToEdit.getMoneyPaid());
+        Money updatedMoneyPerClass = editPersonDescriptor.getMoneyPerClass().orElse(personToEdit.getMoneyPerClass());
```
Comment (cadencjk) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:121]: I think updatedRatesPerClass and getRatesPerClass sounds more natural

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -212,6 +217,18 @@ public Optional<Address> getAddress() {
...
         }
 
+
+        public void setClass(Class aClass) {
+            if (aClass == null) {
+                this.aClass = new Class();
+            } else {
+                this.aClass = aClass;
+            }
+        }
```
Comment (cadencjk) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:227]: May cause conflict with Joan's bug fix #115. Merge with caution.

**Review by cadencjk — COMMENTED** [review]
I feel that refactoring all MoneyPerClass to RatesPerClass could clarify the attribute.

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (Echomo-Xinyu) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:121]: Noted! Will do in the subsequent commit ;)

**Review by cadencjk — APPROVED** [review]
Tested this PR with manual inputs, works well!

LGTM!

**Pull request merged**